src/components/form/range/range.tsx

```diff
@@ -125,12 +125,17 @@ export class EuiRangeClass extends Component<
       (tick) => tick.value.toString() === currentVal.toString()
     );
 
-    if (target) {
+    if (target && !target.accessibleLabel) {
       ariaValueText = target.value.toString();
       return ariaValueText;
-    } else {
-      return ariaValueText; // undefined
     }
+
+    if (target && target.accessibleLabel) {
```

Comment:
Not a change request per se, just a suggestion.
This could be written a bit shorter with a ternary:

```suggestion
if (target) {
      ariaValueText = target?.accessibleLabel
        ? `${target.value.toString()}, (${target.accessibleLabel})`
        : target.value.toString();
    }
```

Reply:
Thank you @mgadewoll. I rewrote this a couple times as a ternary. I didn't like the first pass so I used your solution verbatim because it was much clearer and has the bonus of removing an early `return` clause.